src/runtime_src/core/pcie/driver/linux/xocl/subdev/hwmon_sdm.c

```diff
@@ -50,10 +50,10 @@ struct xocl_sdr_bdinfo {
 	uint64_t mfg_date;
 	uint64_t pcie_info;
 	uint64_t uuid;
-	uint64_t mac_addr0;
-	uint64_t mac_addr1;
-	uint64_t active_msp_ver;
-	uint64_t target_msp_ver;
+	char mac_addr0[SDR_BDINFO_ENTRY_LEN_MAX];
+	char mac_addr1[SDR_BDINFO_ENTRY_LEN_MAX];
```

Comment:
This is ok for now as V70 has only two mac. It is better to take an array of mac addresses and store everything . That way it will be data driven for future.

Reply:
We don't know max macs supported until we receive & parse board info sensor data. So, array of macs may not work directly. Also, VMR is not sending mac_contiguous_num (In XMC, this data used to tell how many macs available).
I will look into any other options to take of your enhancement suggestion in next PR.

Reply:
So, VMR does not tell you the starting MAC and number of contiguous MACs? How does this work? Isn't it a design issue?